packages/tooling/fast-tooling/src/web-components/html-render/html-render.ts

```diff
@@ -119,17 +121,30 @@ export class HTMLRender extends FASTElement {
 
     /// Mouse Handlers
 
+    private getTargetElementFromMouseEvent(e: MouseEvent) {
+        let pathIndex = 0;
+        let el: HTMLElement = e.composedPath()[pathIndex] as HTMLElement;
```

Comment:
You might consider storing `e.composedPath()` in a var so you don't have to keep retrieving it.

Reply:
Done